src/parser/if-then-else.js

```diff
@@ -4,47 +4,53 @@ import type Context from "./context";
 import statement from "./statement";
 import expression from "./expression";
 
-const ifThenElse = (ctx: Context) => {
-  const node = {
-    ...ctx.startNode(ctx.token),
-    then: [],
-    else: []
-  };
-  ctx.eat(["if"]);
-  // First operand is the expression
+// push statement based on then/else branch
+const pushStatement = (node, isThenBranch, stmt) => {
+  if (!stmt) return;
+  if (isThenBranch) return node.then.push(stmt);
```

Comment:
Heads up, I added eslint validation to travis so single line if statements will be a warning from now on. Just try to avoid these.

https://travis-ci.org/ballercat/walt/builds/317486874#L485